openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -38,14 +38,80 @@
                                 href="${ subsection['lms_web_url'] }"
                                 id="${ subsection['id'] }"
                             >
-                                <span>${ subsection['display_name'] }</span>
-                                <span class="sr-only">${ _("This is your last visited course section.") }</span>
-                                % if subsection['current']:
-                                    <span class="resume-right">
-                                        <b>${ _("Resume Course") }</b>
-                                        <span class="icon fa fa-arrow-circle-right" aria-hidden="true"></span>
-                                    </span>
-                                %endif
+                                <div class="subsection-text">
+                                    ## Subsection title
+                                    <span class="subsection-title">${ subsection['display_name'] }</span>
+
+                                    <div class="details">
+                                        ## There are behavior differences between rendering of subsections which have
```

Comment:
It would be good to not mention proctoring in the comments in this file, because there are no special proctoring-related features. I would prefer that this only use one term, but I'm not sure if that is special exams or timed exams. The latter seems better to me, but I don't think I understand the distinction. Maybe it is possible to just call these exams, or is there different behavior for 'non-special' exams?

A lot of this confusion is also in the block API response, so it would be good for us to all get on the same page as to what we're handling here.